port/unix/omrsignal.c

```diff
@@ -1075,7 +1060,7 @@ masterASynchSignalHandler(int signal, siginfo_t *sigInfo, void *contextInfo)
 static uint32_t
 registerSignalHandlerWithOS(OMRPortLibrary *portLibrary, uint32_t portLibrarySignalNo,  unix_sigaction handler)
 {
-	struct sigaction newAction;
+	struct sigaction newAction = {{0}};
```

Comment:
Is this the correct way to do this?  I have seen changes recently I believe to resolve issues on newer XCode or Windows compilers that changed these types of initializers to memset calls.

Reply:
It was for clang compilation on Windows... https://github.com/eclipse/omr/pull/2190/files

Reply:
I assume this code is compiled via clang on OS X and newer versions of clang will warn about this type of initialization

Reply:
replaced `sigaction` initialization with `memset`.